### PR TITLE
User Guide Section 3.0 and 4.0

### PR DESCRIPTION
**Pull request by joey-chance**
Add Section 3.0 FAQ and Section 4.0 Command Summary to User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,192 +1,16 @@
----
-layout: page
-title: User Guide
----
```
Comment (clement0010) [inline docs/UserGuide.md:4]: This part should be retained

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,192 +1,16 @@
...
+## 3.0	FAQ
+Q: How do I transfer my data to another Computer?  
+A: Install the app on the other computer and overwrite the empty data file it creates with the file that contains the data of your previous MedBook folder.  
+## 4.0 Command Summary
```
Comment (clement0010) [inline docs/UserGuide.md:4]:
```suggestion

## 4.0 Command Summary
```

Comment (clement0010) [reply docs/UserGuide.md:4]: Perhaps can add whitespace for readibility

**Comment by clement0010** [comment]
LGTM!

**Review by sibinhho99-nus — APPROVED** [review]

**Pull request merged**